pkg/async-gateway/service.go

```diff
@@ -78,7 +77,8 @@ func (s *service) CreateWorkload(id string, payload io.Reader, headers http.Head
 	}
 
 	log.Debug("sending message to queue")
-	if err := s.queue.SendMessage(id, id); err != nil {
+	queue := NewSQS(queueURL, &s.session)
```

Comment:
Is this efficient / ok to do on each request?

Reply:
Yes, this isn't making any IO calls